service/swagger.js

```diff
@@ -39,7 +39,8 @@ const Team = Joi.object({
   path: Joi.string(),
   users: Joi.array().items(UserRef),
   policies: Joi.array().items(PolicyRef),
-  organizationId: Joi.string()
+  organizationId: Joi.string(),
+  membersCount: Joi.number()
```

Comment:
@mihaidma 

> we should implement count also for single get to have a unified approach

I'm ok with it. I think also with the pagination, having the total number of members could be useful.